EIPS/eip-4626.md

```diff
@@ -115,93 +131,105 @@ The `assetsOf` method SHOULD be used for more accurate calculations of a user's
   inputs: []
 
   outputs:
-  - name: assetsPerUnitShare
-    type: uint256
+    - name: assetsPerShareUnit
+      type: uint256
 ```
 
 #### assetsOf
 
 Total number of underlying assets that `depositor`'s shares represent.
 
-MAY be more accurate than using `assetsPerShare` or `totalAssets / Vault.totalSupply`
-for certain types of fee calculations.
-MAY *NOT* be completely accurate according to slippage or other on-chain conditions.
-MUST *NOT* revert.
+MAY be more accurate than using `assetsPerShare` or `totalAssets / Vault.totalSupply` for certain types of fee calculations.
+
+MAY _NOT_ be completely accurate according to slippage or other on-chain conditions.
+
+MUST _NOT_ revert.
 
 ```yaml
 - name: assetsOf
   type: function
   stateMutability: view
 
   inputs:
-  - name: depositor
-    type: address
+    - name: depositor
+      type: address
 
   outputs:
-  - name: assets
-    type: uint256
+    - name: assets
+      type: uint256
 ```
 
 #### maxDeposit
 
 Total number of underlying assets that `caller` can be deposit.
 
 MUST return a limited value if `caller` is subject to some deposit limit.
+
 MUST return `2 ** 256 - 1` if there is no limit on the maximum amount of assets that may be deposited.
+
 MAY be used in the `previewDeposit` or `deposit` methods for `assets` input parameter.
-MUST *NOT* revert.
+
+MUST _NOT_ revert.
 
 ```yaml
 - name: maxDeposit
   type: function
   stateMutability: view
 
   inputs:
-  - name: caller
-    type: address
+    - name: caller
+      type: address
 
   outputs:
-  - name: maxAssets
-    type: uint256
+    - name: maxAssets
+      type: uint256
 ```
 
 #### previewDeposit
 
 Allows an on-chain or off-chain user to simulate the effects of their deposit at the current block,
 given current on-chain conditions.
 
-MUST return the *exact* amount of Vault shares that would be minted if the caller were to deposit
-a given *exact* amount of underlying assets using the `deposit` method.
+MUST return as close to the _exact_ amount of Vault shares that would be minted if the caller were
```

Comment:
Does it under shoot or overshoot, or is it within some bound of what the correct answer might be? What properties are different versus `assetsPerShare`?